app/controllers/orgs/rosters_controller.rb

```diff
@@ -122,9 +122,25 @@ def add_students
     # rubocop:enable Metrics/AbcSize
 
     def download_roster
+      grouping = Grouping.find_by(id: params[:grouping])
+
+      user_to_groups = {}
+
+      unless grouping.nil?
+        grouping.groups.each do | group|
+          group.repo_accesses.map(&:user_id).each do |id|
+            user_to_groups[id] = group.title
+          end
+        end
+      end
+
       @roster_entries = @current_roster.roster_entries.includes(:user).order(:identifier)
       respond_to do |format|
-        format.csv { send_data @roster_entries.to_csv, filename: "classroom_roster.csv", disposition: "attachment" }
+        format.csv do
+          send_data @roster_entries.to_csv(user_to_groups),
+          filename:    "classroom_roster.csv",
+          disposition: "attachment"
```

Comment:
Unfortunately the linter is still complaining about this 😕 
try writing this as follows:
```ruby
format.csv do
  send_data(
    @roster_entries.to_csv(user_to_groups),
    filename: "classroom_roster.csv",
    disposition: "attachment"
  )
end
```